gateway/src/http_entrypoints.rs

```diff
@@ -1378,6 +1378,25 @@ async fn recovery_host_phase2_upload(
     Ok(HttpResponseOk(HostPhase2RecoveryImageId { sha256_hash }))
 }
 
+/// Get the identifier for the switch this MGS instance is connected to.
+///
+/// Note that most MGS endpoints behave identically regardless of which scrimlet
+/// the MGS instance is running on; this one, however, is intentionally
+/// different.
```

Comment:
Is it worth it to be even more explicit in this docstring? I.e., that this should probably only be used when requesting to `localhost`, or maybe if one is _really sure_ one can map the IP to a physical location.

Reply:
Yeah, that's a good idea. Expanded in d4267b38601022b8d0c2ae660c239e045dea7a47